### ISSUE TITLE
Cannot find module 'klayjs-d3'

### ISSUE THREAD

**Issue by freder** [OpenKieler/klayjs-d3#1]
hi,

I just tried to use `require('klayjs-d3')`, but I get an error.

`test.js`

``` javascript
var d3 = require('d3');
var klayd3 = require('klayjs-d3');
```

``` shell
npm i --save klayjs-d3
node test.js
```

produces:

``` shell
module.js:338
    throw err;
          ^
Error: Cannot find module 'klayjs-d3'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Volumes/Macintosh HD 2/Projects/project/test.js:2:14)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

for vanilla [`klayjs`](https://www.npmjs.com/package/klayjs) it works though.

**Comment by uruuru** [comment]
I guess I forgot to specify the main script in the package.json. I just published a new version 0.3.1. Please give it a try and let me know if it helps.

**Comment by freder** [comment]
works now — thank you.
